packages/jest-enzyme/README.md

```diff
@@ -19,7 +19,8 @@ If you prefer not to use the environment, you can also do this:
 ```js
 // package.json
 "jest": {
-  "setupFilesAfterEnv": ['./node_modules/jest-enzyme/lib/index.js'],
+  "testEnvironment": "enzyme",
```

Comment:
is this something jest supports natively? i'd be very surprised to discover that's the case; it's far more likely that the repo is correct than a 2 year old blog post is.

Reply:
I'm not sure. I was more concerned with the other line being correct because it tripped me up. Looking back at the history, though, I see that the syntax on that [was changed a year ago](https://github.com/FormidableLabs/enzyme-matchers/commit/3eacfea07e28ac8366416e1ae8b6874411c392a0#diff-c00dda8145df11fba3a30a3cab3d425b) to what it is currently. It says that `setupTestFrameworkScriptFile` was deprecated so that's why it was replaced with `setupFilesAfterEnv` but not why. I'll see if I can get it working using that after all.

Reply:
it's likely that jest's format has changed; can you confirm in jest's docs?

Reply:
Yeah actually I was trying to get this to work on a project that's using an older version of Jest, so I'm thinking that it should work as the docs state once we get that update. Will close out.